warp-core/src/main/scala/com/workday/warp/logger/WarpLogUtils.scala

```diff
@@ -53,7 +53,11 @@ object WarpLogUtils extends WarpLogging {
         context.getLogger(loggingLevels.id).setLevel(loggingLevels.level)
       }
 
-      log.getAppender("console").asInstanceOf[ConsoleAppender[ILoggingEvent]].setEncoder(logEncoder)
+      for {
+        maybeAppenders: Appender[ILoggingEvent] <- Seq("console", "CONSOLE") map log.getAppender
```

Comment:
Didn't `find` since `getAppender()` doesn't return a Boolean to use as a predicate, and anything with `flatten` requires this Java method outside my control to return an Option. This seemed to be the route of least redirection, to map then filter in a for comp.